src/main/java/com/amihaiemil/camel/AbstractYamlDump.java

```diff
@@ -32,16 +32,15 @@
  * @author Sherif Waly (sherifwaly95@gmail.com)
  * @version $Id$
  * @since 1.0.0
- * @todo #34:30m/DEV This interface should have 2 more implementor classes:
- *  YamlMapDump and YamlCollectionDump. Each of these classes will encapsulate
- *  and serialize the mentioned types (i.e. Map<Object, Object> and
- *  Collection<Object>). The return type or method ``represent()`` should be
- *  overridden with the proper subtype (e.g. YamlMapDump will have the
- *  method ``YamlMapping represent() {...}``
+ * @todo #34:30m/DEV This interface should have 1 more implementor class:
```

Comment:
@SherifWaly The parent ticket here is 38 :D it also has to be updated so the bot sees it as a new puzzle.